src/AppBundle/Resources/translations/pa-terms-use.en.yml

```diff
@@ -0,0 +1,54 @@
+htmlTitle: Terms of use | GOV.UK
+pageSectionTitle: Terms of use
+section01:
+  heading: |
+    As the public authority deputy, named on the court order, you are responsible
+    for the security of information added to this service.
+  para01: |
+    The public authority (PA) deputy in question, takes responsibility for information
+    assurance relating to the Office of the Public Guardian's (OPG's) digital service
+    for public authority deputies.
+  para02: |
+    Information assurance, in this case, means the strict standards that data protection
+    legislation sets out for dealing with personal information.
+  para03: |
+    For PA deputyships, personal information includes thats gathered about the clients
+    the PA is deputy for, and the user IDs, passwords and email addresses needed to access
+    that information.
+section02:
+  heading: Access to the service
+  para01: |
+    As named deputy you can add a limited number of other colleagues, or users, to help
+    administer the deputyship.
+  para02: |
+    It is then critical that you as public authority named deputy assume sole responsibility for:
```

Comment:
@Biggsen is this rendering properly ? I used to have parsing problems when content with semicolons it not wrapped as misintepreted as a key. If it renders, I guess symfony understands that it's text as under "|"